app.js

```diff
@@ -1,5 +1,79 @@
 // import functions and grab DOM elements
 
-// initialize state
+const play = document.getElementById('play');
+const reset = document.getElementById('reset');
+
+const pcSpan = document.getElementById('pc');
+const winSpan = document.getElementById('wins');
+const lossSpan = document.getElementById('losses');
+const tieSpan = document.getElementById('ties');
+const resetSpan = document.getElementById('reset');
+
+// set initial state
+
+let win = 0;
+let lose = 0;
+let tie = 0;
+let resetIt = 0;
+
+//Object.assign(winSpan, win);
+//Object.assign(lossSpan, lose);
+//Object.assign(tieSpan, tie);
+//Object.assign(resetSpan, resetIt);
+
+console.log(win, lose, tie, resetIt);
+
+// define DOM utitlies
+
+const willYouWin = () => {
```

Comment:
great job, looks like you still need to TDD your youWillwin as well as convert your rpc to getRandomThrow